### PR TITLE
refactor: move legacy namespaces

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/ParcelRegistry/Legacy/Exceptions/ParcelRemovedException.cs [hunk 1]
```diff
@@ -1,10 +1,11 @@
-namespace ParcelRegistry
+namespace ParcelRegistry.Legacy.Exceptions
 {
     using System;
     using System.Runtime.Serialization;
+    using Be.Vlaanderen.Basisregisters.AggregateSource;
 
     [Serializable]
-    public sealed class ParcelRemovedException : ParcelRegistryException
```
Comment (ArneD) [inline src/ParcelRegistry/Legacy/Exceptions/ParcelRemovedException.cs:7]: Why remove this class? We still have it in other registries.

File: test/ParcelRegistry.Tests/InfrastructureEventsTests.cs [hunk 2]
```diff
@@ -1,15 +1,15 @@
-﻿namespace Be.Vlaanderen.Basisregisters.Testing.Infrastructure.Events
+namespace ParcelRegistry.Tests
```
Comment (ArneD) [inline test/ParcelRegistry.Tests/InfrastructureEventsTests.cs:1]: Don't change this, this is generated.

**Review by ArneD — CHANGES_REQUESTED** [review]

File: src/ParcelRegistry/Legacy/Exceptions/ParcelRemovedException.cs (hunk 1 above)
Comment (pgallik) [reply src/ParcelRegistry/Legacy/Exceptions/ParcelRemovedException.cs:7]: We didn't want namespace conflicts with the new domain code, and it wasn't necessary given that there is only one exception in legacy.

**Review by ArneD — APPROVED** [review]

**Pull request merged**